pkg/imageprovider/rhcos_test.go

```
@@ -0,0 +1,177 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package imageprovider
+
+import (
+	"testing"
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)